src/web/handlers/FlingGestureHandler.ts

```diff
@@ -47,7 +47,10 @@ export default class FlingGestureHandler extends GestureHandler {
 
     this.maxNumberOfPointersSimultaneously = 1;
 
-    this.delayTimeout = setTimeout(() => this.fail(), this.maxDurationMs);
+    this.delayTimeout = setTimeout(
+      () => this.fail(),
+      this.maxDurationMs
+    ) as unknown as number;
```

Comment:
I think I already wrote it somewhere, but I'm not sure.

Wouldn't `ReturnType<typeof setTimeout>` as delayTimeout work?

Reply:
Done in [f6a46e5](https://github.com/software-mansion/react-native-gesture-handler/pull/2873/commits/f6a46e59bf7ebe489416d68fc2700e18d5bb3d05)